spec/features/admin/enrollments/edit_spec.rb

```diff
@@ -5,40 +5,60 @@
     create(:enrollment, location: 'Eloise May')
   end
 
-  before do
-    visit edit_admin_enrollment_path(eloise_may)
+  let(:date_field_css) do
```

Comment:
Absolutely love this!

<img src="https://media1.giphy.com/media/3q3QK6KyDVUBzih7hB/giphy.gif"/>